src/measure_theory/vector_measure.lean

```diff
@@ -618,6 +621,33 @@ begin
   { rw [v.not_measurable hi, w.not_measurable hi] }
 end
 
+lemma restrict_le_restrict_iff (v w : vector_measure α M) {i : set α} (hi : measurable_set i) :
+  v.restrict i ≤ w.restrict i ↔ ∀ ⦃j⦄, measurable_set j → j ⊆ i → v j ≤ w j :=
+⟨λ h j hj₁ hj₂, (@restrict_eq_self _ _ _ _ _ v _ hi _ hj₁ hj₂) ▸
+  (@restrict_eq_self _ _ _ _ _ w _ hi _ hj₁ hj₂) ▸ h j hj₁,
+ λ h, le_iff.1 (λ j hj, (@restrict_apply _ _ _ _ _ v i hi j hj).symm ▸
+  (@restrict_apply _ _ _ _ _ w i hi j hj).symm ▸ h (hj.inter hi) (set.inter_subset_right j i))⟩
+
+lemma subset_le_of_restrict_le_restrict (v w : vector_measure α M) {i : set α}
+  (hi : measurable_set i) (hi₂ : v.restrict i ≤ w.restrict i) : ∀ ⦃j⦄, j ⊆ i → v j ≤ w j :=
+begin
+  intros j hj,
```

Comment:
Probably better before the colon:
```suggestion
  (hi : measurable_set i) (hi₂ : v.restrict i ≤ w.restrict i) ⦃j : set α⦄ (hj : j ⊆ i) :
  v j ≤ w j :=
begin
```
I suspect you're not gaining any benefit from the ` ⦃j : set α⦄` vs just `{j : set α}` here, but I haven't looked carefully.